host_core/lib/host_core/control_interface/server.ex

```diff
@@ -269,4 +291,18 @@ defmodule HostCore.ControlInterface.Server do
       error: error
     })
   end
+
+  defp publish_host_stopped(host_id) do
+    prefix = HostCore.Host.lattice_prefix()
+
+    msg =
+      %{
```

Comment:
You can actually supply an empty map here, because the host is always provided as the `source` field on the CloudEvent, per this code https://github.com/wasmCloud/wasmcloud-otp/blob/main/host_core/lib/host_core/cloud_event.ex#L4